pumpkin/src/world/mod.rs

```diff
@@ -2380,6 +2386,180 @@ impl World {
         .await;
     }
 
+    pub fn queue_decrease(self: &Arc<Self>, pos: BlockPos, level: u8) {
```

Comment:
What a great day to start documenting functions! (Also should probably have a more descriptive name to go along with it)